src/esp/gfx/PbrShader.cpp

```diff
@@ -359,6 +372,33 @@ PbrShader& PbrShader::bindPrefilteredMap(Magnum::GL::CubeMapTexture& texture) {
   return *this;
 }
 
+PbrShader& PbrShader::bindPointShadowMap(int index,
+                                         Magnum::GL::CubeMapTexture& texture) {
+  CORRADE_ASSERT(
+      index >= 0 && index < 3,
+      "PbrShader::bindPointShadowMap(): the texture index was illegal.", *this);
+  CORRADE_ASSERT(flags_ & Flag::ShadowsVSM,
+                 "PbrShader::bindPointShadowMap(): the shader was not "
+                 "created with shadows enabled",
+                 *this);
+  switch (index) {
+    case 0:
+      texture.bind(pbrTextureUnitSpace::TextureUnit::ShadowMap0);
+      break;
+    case 1:
+      texture.bind(pbrTextureUnitSpace::TextureUnit::ShadowMap1);
+      break;
+    case 2:
+      texture.bind(pbrTextureUnitSpace::TextureUnit::ShadowMap2);
+      break;
+
+    default:
+      CORRADE_INTERNAL_ASSERT_UNREACHABLE();
+      break;
+  }
```

Comment:
```suggestion
  texture.bind(pbrTextureUnitSpace::TextureUnit::ShadowMap0 + index);
```

The assertion above does enough work to ensure this is always correct ;)